client/src/components/menubar/index.js

```diff
@@ -0,0 +1,693 @@
+                marginLeft: 10
+              }}
+            >
+              <Tooltip content="Undo" position="left">
+                <AnchorButton
+                  type="button"
+                  className="bp3-button bp3-icon-undo"
+                  disabled={undoDisabled}
+                  onClick={() => {
+                    dispatch({ type: "@@undoable/undo" });
+                  }}
+                  style={{
+                    cursor: "pointer"
+                  }}
+                />
+              </Tooltip>
+              <Tooltip content="Redo" position="left">
+                <AnchorButton
+                  type="button"
+                  className="bp3-button bp3-icon-redo"
+                  disabled={redoDisabled}
+                  onClick={() => {
+                    dispatch({ type: "@@undoable/redo" });
+                  }}
+                  style={{
+                    cursor: "pointer"
+                  }}
+                />
+              </Tooltip>
+            </div>
+            <Tooltip
+              content="Reset cellxgene, clearing all selections"
+              position="left"
+            >
+              <AnchorButton
+                disabled={this.isResetDisabled()}
+                style={{ marginLeft: 10 }}
+                type="button"
+                loading={resettingInterface}
+                intent="none"
+                icon="refresh"
+                onClick={this.resetInterface}
+                data-testid="reset"
+                data-testclass={`resetting-${resettingInterface}`}
+              >
+                reset
+              </AnchorButton>
+            </Tooltip>
+
+            <div style={{ marginLeft: 10 }} className="bp3-button-group">
+              <Popover
+                content={
+                  <Menu>
+                    <MenuItem
+                      href="https://chanzuckerberg.github.io/cellxgene/faq.html"
+                      target="_blank"
+                      icon="help"
+                      text="FAQ"
+                    />
+                    <MenuItem
+                      href="https://join-cellxgene-users.herokuapp.com/"
+                      target="_blank"
+                      icon="chat"
+                      text="Chat"
+                    />
+                    <MenuItem
+                      href="https://chanzuckerberg.github.io/cellxgene/"
+                      target="_blank"
+                      icon="book"
+                      text="Docs"
+                    />
+                    <MenuItem
+                      href="https://github.com/chanzuckerberg/cellxgene"
+                      target="_blank"
+                      icon="git-branch"
+                      text="Github"
+                    />
+                    <MenuItem
+                      target="_blank"
+                      text={`cellxgene v${
+                        libraryVersions && libraryVersions.cellxgene
```

Comment:
when I test, the version number does not show up in the menu.    It displays as "vnull"

Reply:
the property is not present in the state `@connect`